zbus/src/connection/socket/unix.rs

```diff
@@ -348,7 +349,21 @@ fn get_unix_peer_creds_blocking(fd: RawFd) -> io::Result<crate::fdo::ConnectionC
             )
         };
 
+        let mut groups = User::from_uid(uid)?
+            .map(|user| CString::new(user.name))
+            .transpose()?
+            .map(|user| getgrouplist(&user, gid))
+            .transpose()
+            .ok()
+            .flatten()
+            .unwrap_or(Vec::new());
+        groups.sort_by_key(|gid| gid.as_raw());
```

Comment:
Not sure we want to do any sorting here unless the spec recommends or requires it.

Reply:
The spec requires it. Added a comment.